modules/osd-applications-config-custom-domains.adoc

```diff
@@ -45,12 +45,14 @@ metadata:
 spec:
   domain: apps.companyname.io <1>
   scope: External
+  loadBalancerType: NLB <2>
```

Comment:
This can be either "NLB" or "Classic". "Classic" should be the default, not "NLB".